about.html

```diff
@@ -12,60 +12,84 @@
+            </div>
+        </section>
 
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
 
+    <!-- grid middle -->
 
+    <section id="aboutMain">
+      <div class="aboutGrid">
+        <img src="img/about-plan.png" alt="strategy">
+        <h2>Strategy</h2>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <a href="#">Learn More</a>
+      </div>
 
-    <img src="img/about-office.png" alt="strategy">
+      <div class="aboutGrid">
+        <img src="img/about-working.png" alt="strategy">
+        <h2>How We Work</h2>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <a href="#">Learn More</a>
+        
+      </div>
     
-    Places We Work
+      <div class="aboutGrid">
+       <img src="img/about-office.png" alt="strategy">
+        <h2>Places We Work</h2>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <a href="#">Learn More</a>
+        
+      </div>
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <div class="aboutGrid">
+        <img src="img/about-meeting.png">
+        <h2>Collaboration</h2>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <a href="#">Learn More</a>
+      </div>
+    </section>
 
-    Learn More
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
 
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+    <section class="aboutFooter">
+      <div class="about-bottom-content">
+        <h3>Let's Work Together</h3>
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+        <br>
```

Comment:
Try avoiding break tags when you can! Use your CSS knowledge to place and space items.